packages/edit-post/src/components/header/more-menu/index.js

```diff
@@ -22,11 +27,13 @@ const TOGGLE_PROPS = {
 	tooltipPosition: 'bottom',
 };
 
-const MoreMenu = () => (
+const MoreMenu = ( { showIconLabel } ) => (
 	<DropdownMenu
-		className="edit-post-more-menu"
+		className={ classnames( 'edit-post-more-menu', {
+			'show-icon-label': showIconLabel,
+		} ) }
 		icon={ moreVertical }
-		label={ __( 'More tools & options' ) }
+		label={ __( 'More options' ) }
```

Comment:
@mapk, as per your last [comment](https://github.com/WordPress/gutenberg/pull/15830#issuecomment-528135873), I've trimmed this down a bit from the source.

Reply:
the reason I've removed `tools`, but not `options` is because `'More options` makes more sense than `'More tools` in context of the existing content of that dropdown, correct me if I am wrong